src/markdown-pages/instant-observability/build-a-quickstart/create-a-quickstart.mdx

```diff
@@ -0,0 +1,482 @@
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE 
+
+# List of Critical and Warning thresholds for the condition
+terms:
+- priority: CRITICAL
+  # Operator used to compare against the threshold.
+  operator: ABOVE
+  # Value that triggers a violation
+  threshold: 0.85
+  # Time in seconds; 120 - 3600
+  thresholdDuration: 300
+  # How many data points must be in violation for the duration
+  thresholdOccurrences: ALL 
+
+# Adding a Warning threshold is optional
+- priority: WARNING
+  operator: ABOVE 
+  threshold: 0.75
+ thresholdDuration: 300
+  thresholdOccurrences: ALL 
+
+# Duration after which a violation automatically closes
+# Time in seconds; 300 - 2592000 (Default: 86400 [1 day])
+violationTimeLimitSeconds: 259200
+```
+
+## Add a description
+
+Your quickstart is almost ready to go live. But how will other's know and understand what your quickstart is about and what it can do?
+To help others understand the purpose of your quickstart, add a description. 
+
+In the root directory of your pack, you can find a _config.yml_ file. 
+Open the file in your IDE and edit it as follow:
+
+```yml
+ 
+# Name of the pack (required)
+name: flashdb
+
+# Description of the pack (required)
+description: |
+  Flashdb is the fastest database in the space. 
+  The flashdb quickstart allows you to get visibility into the performance of flashdb with the help of following features:
+
+  Dashboards:
+  - Database methods: Monitor the number of transactions for each database method
+  - Database size: Monitor the size of database
+  - Average response time: Monitor the average response time for each type of database transaction
+  - Cache hits: Monitor the cache hits
+  - Errors: Monitor the errors occured for each type of database transaction
+  - Database keys: Monitor the keys for database
+ 
+  Alerts:
+  - Cache hit ratio: Alert is triggered when cache hit ratio falls below the set limit.
+  - Read response: Alert is triggered when read responses are slow and exceeds a set limit
+
+summary: |
+  Flashdb is the fastest database in the space. 
+  This quickstart allows you to get visibility into the performance of flashdb.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Design
+icon: icon.jpeg
+logo: logo.png
+
+# Authors of the pack (required)
+authors:
+  - {"enter your name"}
+
+# References to other packs (optional)
+# References are grouped by the type: operating-system, database, webserver, queue, ..
+# The reference under the type is a at least one system, so at least one operating-system, database, ..
+references:
+  - type: database
+    name:
+    - flashdb
+```
+
+Here's how your final _flashdb_ quickstart folder should look like.
+![flashdb quickstart directory](../../../images/instant-observability/final-flashdb-quickstart.png)
+
+## Test your quickstart
```

Comment:
While the import script is not working, let's remove this section. Once the import script functions appropriately, it can be saved in a branch and added back into the lab.